developers/weaviate/search/bm25.md

```diff
@@ -122,6 +132,24 @@ You can retrieve the BM25F `score` values for each returned object.
     />
   </TabItem>
 
+  <TabItem value="go" label="Go">
```

Comment:
This block seems to be duplicated. Would you delete lines 144-151?

Reply:
sure, no problem